src/bci_build/package/keadhcp/README.md.j2

```diff
@@ -0,0 +1,36 @@
+# {{ image.pretty_name }} container image
+
+{% include 'badges.j2' %}
+
+ {{ image.description }}
```

Comment:
leading whitespace
```suggestion
{{ image.description }}
```

Alternatively I'd suggest a more verbose description, e.g.:
```
Kea is an open-source DHCP server developed by the [Internet Systems
Consortium](https://www.isc.org/) and the successor of the now deprecated
DHCPd. The Kea distribution includes a DHCPv4 server, a DHCPv6 server, and a
Dynamic DNS (DDNS) server. Significant features include: support for IPv6 prefix
delegation, host reservations (which may be optionally stored in a separate back
end database), Preboot Execution Environment (PXE) boot, client classification,
shared networks, and high-availability (failover pairs). Kea can store leases
locally in a memfile, or in a PostgreSQL or MySQL database. Kea has a supported
API for writing optional extensions, using 'hooks'.
```
(yes, that's mostly from Wikipedia)